apps/cookbook/src/app/examples/form-field-example/form-field-example.component.ts

```diff
@@ -9,7 +9,10 @@ import { WindowRef } from '@kirbydesign/designsystem/types';
 })
 export class FormFieldExampleComponent {
   constructor(private window: WindowRef) {}
-
+  size: string = 'lg';
```

Comment:
Consider using `InputSize`
```suggestion
  size: InputSize = InputSize.Large;
```